### ISSUE TITLE
code convention?

### ISSUE THREAD

**Issue by disjukr** [heejongahn/ts-for-jsdev#4]
2space를 사용하는 곳이 있고 4space를 사용하는 곳도 있네요.
[한 페이지 안에서 두 컨벤션이 공존하는 경우](https://ahnheejong.gitbook.io/ts-for-jsdev/03-basic-grammar/function)도 보이고, [한 예시 안에서 공존하는 경우](https://ahnheejong.gitbook.io/ts-for-jsdev/01-introducing-typescript/why-typescript#undefined-1)도 보입니다.
가독성을 위해서 통일하는게 좋을 것 같습니다.

**Comment by heejongahn** [comment]
네 예제코드를 그때그때 작성해서 확인하는 정도로만 작성하고 크게 신경을 쓰지 못한 것이 사실입니다... 여력이 생기면 반영하도록 하겠습니다.

**Comment by marocchino** [comment]
[diff.txt](https://github.com/heejongahn/ts-for-jsdev/files/2019945/diff.txt)

그냥 다른거 하는 김에 생각나서 적당히 프리터 돌려서 디프 쪄왔습니다.
근데 고쳐지 말아야 할 부분을 고친것도 보이네요.
참고용으로만..
